scout/server/blueprints/institutes/views.py

```diff
@@ -138,7 +138,8 @@ def causatives(institute_id):
     variants = list(store.check_causatives(institute_obj=institute_obj, limit_genes=hgnc_id))
     if variants:
         variants = sorted(
-            variants, key=lambda k: k.get("hgnc_symbols", [None])[0] or k.get("str_repid")
+            variants,
+            key=lambda k: k.get("hgnc_symbols", [None])[0] or k.get("str_repid") or "",
```

Comment:
Looking good

Reply:
Yeah, that is showing in the screen-shot above! 😊

Reply:
Ah OK sorry, it was kind of obvious!